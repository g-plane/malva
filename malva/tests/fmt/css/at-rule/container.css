@container my-container scroll-state( scrollable:top ) {}
