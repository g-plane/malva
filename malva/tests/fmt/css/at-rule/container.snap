---
source: malva/tests/fmt.rs
---
@container my-container scroll-state(scrollable: top) {}
